src/algorithms.ts

```diff
@@ -29,6 +31,82 @@ export const getKtyFromAlg = (alg: string) => {
   }
 };
 
+/**
+ * Given a key type, valdates if a signing algorithm is compatible
+ *
+ * @param kty The given key type
+ * @param alg The signing algorithm
+ *
+ * @returns The provided algorithm in case it's compatible
+ * @throws {JOSENotSupported} If the signing algorithm is not compatible with the provided key type
+ */
+export const validateAlgFromKty = (
```

Comment:
Need unit test!

Reply:
Done in https://github.com/pagopa/io-react-native-jwt/pull/21/commits/327f53cb257d6aa1857bfbe97e79cb7be16b7028. Also `validateAlgFromKty` wasn't actually used, I removed it.